### PR TITLE
Add GCP Rotate Keys orb

### PR DESCRIPTION
**Pull request by chris-brindley**
This PR adds a new orb to the repository called `gcp-rotate-keys`. It is designed to help teams rotate their GCP service account credentials.

In my team, our use case involves adding the new key to SSM and redeploying a Fargate service before deleting the old key, in order to prevent any downtime. We were unable to fulfil this requirement with existing tools (e.g. the cloud key rotator) so I have written an orb that is hopefully re-usable by others as well.

The orb has two commands - `rotate-gcp-key` which takes a series of steps to perform actions after creating the new key but before deleting the old one, and `rotate-gcp-key-in-aws-ssm` which will upload the key in AWS SSM Parameter Store before running the series of steps.

It also includes a single job - `rotate-key-redeploy-ecs-service` which implements our use case of redeploying a service on Fargate during the key rotation to pick up the new key.

### CODE REVIEW

**Review by gregorath — APPROVED** [review]

File: gcp-rotate-keys/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+* `google-project-id` - The Google project ID to connect with
+* `redeploy-timeout` - Number of seconds to wait for the redeployment to reach
+  steady state
+* `service-account` - The service account email address to rotate the key of
+* `ssm-path` - The SSM path where the service account key is saved
+
+## Limitations
+
+Currently the commands will abort without doing anything if the service account
+has more than key.
```
Comment (eversC) [inline gcp-rotate-keys/README.md:81]:
```suggestion
has more than one key.
```

**Review by eversC — CHANGES_REQUESTED** [review]
Looks good. Probably the smallest request for change possible from me :D

**Review by enicholson125 — APPROVED** [review]
Looks good to me!

**Review by eversC — APPROVED** [review]
lgtm

Orb publishing is currently blocked by auth failures unfortunately, just fyi

**Pull request merged**